packages/context-menu/src/vaadin-menu-overlay-mixin.js

```diff
@@ -58,15 +75,6 @@ export const MenuOverlayMixin = (superClass) =>
       });
     }
 
-    /**
-     * Returns the first element in the overlay content.
-     *
-     * @returns {HTMLElement}
-     */
-    getFirstChild() {
```

Comment:
I'm not sure why this is in the public API, but it wasn't really that useful in the context of this component.